SwiftGit2/Pointers.swift

```diff
@@ -45,27 +45,29 @@ public enum Pointer: PointerType {
 	
 	public var type: git_otype {
 		switch self {
-		case let Commit(oid):
+		case Commit(oid):
 			return GIT_OBJ_COMMIT
-		case let Tree(oid):
+		case Tree(oid):
 			return GIT_OBJ_TREE
-		case let Blob(oid):
+		case Blob(oid):
 			return GIT_OBJ_BLOB
-		case let Tag(oid):
+		case Tag(oid):
 			return GIT_OBJ_TAG
+		default:
+			return GIT_OBJ_ANY
```

Comment:
This default case isn't needed. I suspect Swift is getting confused about what `Commit`, etc. mean here. Since the `let` was removed, the variables should also be removed:

```swift
	public var type: git_otype {
		switch self {
		case Commit:
			return GIT_OBJ_COMMIT
		case Tree:
			return GIT_OBJ_TREE
		case Blob:
			return GIT_OBJ_BLOB
		case Tag:
			return GIT_OBJ_TAG
		}
	}
```